### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by ark0f**

### CODE REVIEW

File: scripts/src/init.sh [hunk 1]
```diff
@@ -56,7 +56,7 @@ cargo_init() {
   if [ -z $CI ]; then
     cargo install cargo-hack
     cargo install cargo-nextest
-  elif [ "$OSTYPE" == "linux-gnu" ]; then
+  elif [ "$OSTYPE" = "linux-gnu" ]; then
```
Comment (shamilsan) [inline scripts/src/init.sh:59]: Please check it twice

File: .github/workflows/CI.yaml [hunk 2]
```diff
@@ -335,7 +335,7 @@ jobs:
           path: artifact
 
   build-win:
-    if: github.ref == 'refs/heads/master'
+    #if: github.ref == 'refs/heads/master'
```
Comment (shamilsan) [inline .github/workflows/CI.yaml:338]: Do we really want to run Win CI in PR?

Comment (shamilsan) [reply .github/workflows/CI.yaml:338]: Ah, got it, it is a check.

File: scripts/src/init.sh (hunk 1 above)
Comment (ark0f) [reply scripts/src/init.sh:59]: We are because after Substrate update `protoc` is required to be installed

Comment (ark0f) [reply scripts/src/init.sh]: I'm trying and everything is successful even in PR CI check, but not in master. There is an another try

**Comment by ark0f** [comment]
Successful runs:
| OS | URL |
| --- | --- |
| macOS | https://github.com/gear-tech/gear/actions/runs/3084673468/jobs/4987100014 |
| Windows | https://github.com/gear-tech/gear/actions/runs/3084673468/jobs/4987100285 |

**Review by shamilsan — APPROVED** [review]

**Pull request merged**